app/helpers/application_helper.rb

```diff
@@ -1,5 +0,0 @@
-module ApplicationHelper
-  def feedback_address
-    Maslow::Application.config.feedback_address
-  end
-end
```

Comment:
Good find. Looks like this was added in 57cbb39e8db1af9c08dd4f9be7bf2d547e18078c and (partially) deleted in 1b7e59f1f1a1862f234d91d07226c671ea77e836.